content/en/getting-started/help-and-support/index.md

```diff
@@ -10,89 +10,120 @@ cascade:
-- [Report existing bugs](https://github.com/localstack/localstack/issues/new?assignees=&labels=type%3A+bug%2Cstatus%3A+triage+needed&template=bug-report.yml&title=bug%3A+%3Ctitle%3E)
+You can avail community support through the following channels:
 
-Make sure to follow the issue templates and provide as much information as possible.
-If you have encountered outdated documentation, please report it on our [documentation GitHub page](https://github.com/localstack/docs).
+- [LocalStack Discuss](https://discuss.localstack.cloud/)
+- [LocalStack Slack Community](https://localstack.cloud/slack)
+- [GitHub Issue](https://github.com/localstack/docs/issues/new)
 
-## Dedicated support
+Community support is provided on a best-effort basis and is not guaranteed.
+We also encourage you to help others in the community by answering questions and sharing your experiences.
 
-Dedicated customer support is available to **Starter** & **Teams** plan users with an active subscription.
+### Standard support
 
-The support team is available to help you with:
+Standard support is available to all LocalStack users with an active Starter or Trial subscription.
+With standard support, you get access to 1v1 support based on best effort basis.
+With the best effort support, we do not provide fixed response times.
+However, as we understand the importance of timely assistance, our support team always strives to respond to inquiries and requests within 24–48 hours during our [regular business hours](#support-business-hours).
 
-- Troubleshooting LocalStack issues
-- Answering questions about LocalStack features and functionality
-- Providing guidance on how to use LocalStack
-- Providing guidance on how to integrate LocalStack with your application
+#### Troubleshooting Guidelines
 
-However, dedicated support does not include:
+In case you encounter any issues with LocalStack, follow these troubleshooting guidelines:
 
-- Support for AWS services
-- General troubleshooting of your application
-- Billing, account issues, or other questions that are not technical
+- Review the documentation and FAQs for possible solutions.
+- Check the system requirements to ensure compatibility.
+- Verify the configuration settings against the documentation.
+- Consult the error logs for any relevant error messages or warnings.
+- Seek guidance from the community through the user forum.
+
+{{< callout "note" >}}
+In many scenarios, we ask our customers to use Diagnosis endpoint to help us retrieve additional information.
+
+To use LocalStack's Diagnosis endpoint:
+
+- Set the environment variable `LS_LOG=trace`
+- Start LocalStack
+- Run the affected task(s)
+- Call the diagnostic endpoint `curl -s localhost:4566/_localstack/diagnose | gzip -cf > diagnose.json.gz` (Endpoint URL depends on your configuration)
+- Send the output file to our Slack chat
+
+Ensure that you avoid sending the diagnostic output to public channels or forums, as it may contain sensitive information.
```

Comment:
```suggestion
<span style="color: darkred;">**Ensure that you avoid sending the diagnostic output to public channels or forums, as it may contain sensitive information.**</span>
```
I would like to emphasize this sentence even more.